istio-control/istio-discovery/files/injection-template.yaml

```diff
@@ -192,6 +192,8 @@ template: |
       {{ toYaml .Values.global.proxy.lifecycle | indent 4 }}
     {{- end }}
     env:
+    - name: SIGN_CERT_AT_KUBERNETES_CA
```

Comment:
this one is not needed though, we can just detect based on file, right? otherwise lgtm

Reply:
Yes, this one is not strictly needed and the detection will be based on the existence of the file under a special directory. I have removed this one from the PR.